specification/maps/data-plane/Creator/preview/2022-09-01-preview/mapconfiguration.json

```diff
@@ -350,44 +360,66 @@
           "description": "The description the caller provided when creating the map configuration. Maximum length 1024 characters.",
           "type": "string"
         },
-        "defaultStyle": {
-          "description": "The default style for the map configuration. The value can be one of the 'name' field in StyleObject under styles array.",
+        "defaultConfiguration": {
+          "description": "The default configuration for the map configuration. The value should be one of the 'name' field in ConfigurationObject under styles array. If not provided, first entry in the 'Configurations' list will be used.",
           "type": "string"
         },
-        "styles": {
-          "description": "Styles array will allow customer to configure different map representations for their tileset. Each style configured here will be reflected as one style in style picker in client SDK. The name should be unique in the styles array.",
+        "configurations": {
+          "description": "Configurations array will allow customer to configure different map representations for their tileset. Each configuration configured here will be reflected as one map style in style picker in client SDK. The name should be unique in the configurations array.",
           "items": {
-            "$ref": "#/definitions/StyleObject"
+            "$ref": "#/definitions/ConfigurationObject"
           }
         }
       }
     },
-    "StyleObject": {
+    "ConfigurationObject": {
       "description": "Define the format to configure a style picker and the associated map in client SDK.",
       "type": "object",
+      "required": [
+        "name",
+        "layers"
+      ],
       "properties": {
         "name": {
-          "description": "The name for the style.",
+          "description": "The name for the style. Maximum length 255 characters.",
           "type": "string"
         },
         "displayName": {
-          "description": "The display name for the style picker in client SDK.",
+          "description": "The display name for the style picker in client SDK. Maximum length 255 characters.",
           "type": "string"
         },
         "description": {
-          "description": "The description for the style. Maximum length 1024 characters.",
+          "description": "The description for the style. Maximum length 500 characters.",
           "type": "string"
         },
         "theme": {
-          "description": "The theme for client SDK widgets(style picker, zoom, level).",
-          "type": "string"
+          "description": "The theme for client SDK widgets(style picker, zoom, level picker). Maximum length 100 characters.",
```

Comment:
This description doesn't seem to match the enum values defined. Also, it makes little sense to say "Maximum length 100 characters" when the largest enum value is only 5 characters.

Reply:
Updated the description and removed the 100 characters length.